app/templates/views/user-profile/change.html

```diff
@@ -20,9 +20,20 @@
     <div class="govuk-grid-column-three-quarters">
       {% call form_wrapper() %}
         {{ form_field(error_message_with_html=True) }}
-        {{ page_footer('Save') }}
+        {% if current_user.auth_type == 'email_auth' %}
```

Comment:
```suggestion
        {% if current_user.auth_type != 'sms_auth' %}
```

This is inconsistent with the conditional in the endpoint.

Reply:
You are right it is inconsistent, I would change it the other way round - as people who use webauthn should still have their phone numbers as a backup for if they lose the key we downgrade them to sms auth if I remember the process around this correctly.